types/git-semver-tags/git-semver-tags-tests.ts

```diff
@@ -1,13 +1,17 @@
-"use strict";
-
-import gitSemverTags from "git-semver-tags";
+import gitSemverTags = require('git-semver-tags');
```

Comment:
Will this be better?

```suggestion
import * as gitSemverTags from "git-semver-tags";
```